Sources/Sentry/SentryScope.m

```diff
@@ -123,8 +123,20 @@ - (void)addBreadcrumb:(SentryBreadcrumb *)crumb
     if (self.maxBreadcrumbs < 1) {
         return;
     }
-    [SentryLog logWithMessage:[NSString stringWithFormat:@"Add breadcrumb: %@", crumb]
-                     andLevel:kSentryLevelDebug];
+
+    // since we will create a breadcrumb for each log statement, we won't log breadcrumbs with
+    // SentryLog or else we'll end up in an infinite loop. use printf instead so we can still see
+    // this in the console.
```

Comment:
How so? I never saw this happening, breadcrumb for log statements.

Reply:
Ah, that's part of the new functionality added in this PR. But, we _do_ write to SentryLog for every breadcrumb that's written. That's why I had to make the change you're looking at, because doing so created the infinite loop.

That being said, I'm not seeing all the breadcrumbs in the Sentry dashboard. I'm still trying to figure out why.

Reply:
I'm removing the breadcrumb aspect of this PR.